From 38175614ecb809f49f52be6ab5d6684c06337de2 Mon Sep 17 00:00:00 2001
From: P Praneesh <quic_ppranees@quicinc.com>
Date: Fri, 29 Apr 2022 18:37:04 +0530
Subject: [PATCH] ath12k: Fix firmware assert due to ce count mismatch

Due to target CE count mismatch, observed firmware crash
with the latest SU. Fix it by correcting the target CE count.

Signed-off-by: P Praneesh <quic_ppranees@quicinc.com>
---
 drivers/net/wireless/ath/ath12k/ce.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/drivers/net/wireless/ath/ath12k/ce.c b/drivers/net/wireless/ath/ath12k/ce.c
index be0d669..b9fae1c 100644
--- a/drivers/net/wireless/ath/ath12k/ce.c
+++ b/drivers/net/wireless/ath/ath12k/ce.c
@@ -140,6 +140,14 @@ const struct ce_attr ath12k_host_ce_config_qcn9274[] = {
 		.src_sz_max = 0,
 		.dest_nentries = 0,
 	},
+
+	/* CE15: reserved for customer use */
+	{
+		.flags = (CE_ATTR_FLAGS | CE_ATTR_DIS_INTR),
+		.src_nentries = 0,
+		.src_sz_max = 0,
+		.dest_nentries = 0,
+        },
 };
 
 const struct ce_attr ath12k_host_ce_config_wcn7850[] = {
-- 
2.17.1

